octobot_tentacles_manager/api/inspector.py

```diff
@@ -26,6 +26,26 @@ def get_installed_tentacles_modules() -> set:
     return set(tentacle for tentacle in loaders.get_tentacle_classes().values())
 
 
+def get_installed_tentacles_modules_dict() -> dict:
```

Comment:
What will it be used for?

Reply:
this can be used with the react ui to manage installed tentacles
meanwhile i moved it to the react ui

I also updated the pull request - I removed create folder if it doesnt exist as its already fixed in master

Reply:
Why can't you use `get_installed_tentacles_modules` to do it?

Reply:
its returning as a set and tentacle_type is an object which doesnt work with json without changing it

Reply:
Why not converting to json in client (caller) side?